### DISCUSSION TITLE
How can I run multiple msfconsole session without affecting the current one?

### DISCUSSION THREAD

**Discussion by flamecopper** [rapid7/metasploit-framework#16277] [General]
For example, I just need to open multiple windows in console:
hydra -V -L Usernames.txt -P Passwords.txt $ip smb -t 30 -T 100 -I -s 445
However in metasploit I need to go through the process of opening msfconsole in every single windows.
How can I run taskes in parallel without affecting the current one?

**Comment by smcintyre-r7** [comment]
This sounds like a problem that could be addressed with some terminal configuration. I like [terminator](https://terminator-gtk3.readthedocs.io/en/latest/) alot and it can be configured with different profiles which you could use to automatically start msfconsole if you wanted. It also lets windows be "grouped" so you can select multiple terminal windows, enter commands in one and have all of the terminals get receive them. It's great if you're trying to configure the same module in multiple metasploit instances.

I'm sure there's other ways to solve this problem as well, terminator is just my favorite.
